src/sql/workbench/contrib/assessment/test/common/vsServicesStubs.ts

```diff
@@ -0,0 +1,205 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { IEnvironmentService, IExtensionHostDebugParams } from 'vs/platform/environment/common/environment';
+import { URI } from 'vs/base/common/uri';
+import * as files from 'vs/platform/files/common/files';
+import { IDisposable } from 'vs/base/common/lifecycle';
+import { VSBuffer, VSBufferReadable, VSBufferReadableStream } from 'vs/base/common/buffer';
+import { Event } from 'vs/base/common/event';
+import { IDialogService, IConfirmation, IDialogOptions, IConfirmationResult, IShowResult } from 'vs/platform/dialogs/common/dialogs';
+import Severity from 'vs/base/common/severity';
+import { IClipboardService } from 'vs/platform/clipboard/common/clipboardService';
+import {
+	INotificationService,
+	INotification,
+	INotificationHandle,
+	IPromptOptions,
+	IStatusMessageOptions,
+	NotificationsFilter,
+	NotificationMessage,
+	IPromptChoice
+} from 'vs/platform/notification/common/notification';
+
+
+export class EnvironmentServiceStub implements IEnvironmentService {
```

Comment:
You can't use https://github.com/microsoft/azuredatastudio/blob/main/src/vs/workbench/test/browser/workbenchTestServices.ts#L31 ?1

Reply:
Yes. I'm getting the error:
`
Reading git index versions...
..\azuredatastudio\src\sql\workbench\contrib\assessment\test\common\asmtActions.test.ts: line 34, col 33, Warning - Bad layering. You are not allowed to access browser from here, allowed layers are: [common] (code-layering)
`

Reply:
I'd suggest moving both the actions and tests into the browser layer then (`workbench/contrib/assessment/browser` and `workbench/contrib/assessment/test/browser`. Given that the component is in the browser anyways that shouldn't prove to be a problem and not having to duplicate all this code would be great.